lobby-db-dao/src/main/java/org/triplea/lobby/server/db/dao/user/ban/UserBanDao.java

```diff
@@ -28,27 +28,40 @@
           + " desc")
   List<UserBanRecord> lookupBans();
 
+  // TODO: Project#12 convert select to be an update to count matches against system id vs IP
+  // TODO: test that expired bans are not returned when IP address or system id does match.
```

Comment:
I want to re-work this query in a follow-up; so it's made correct now,  but not yet tested. I'll update the query and tet code in a focused follow-up.